code/go/0chain.net/blobbercore/datastore/postgres.go

```diff
@@ -18,6 +17,17 @@ type postgresStore struct {
 	db *gorm.DB
 }
 
+func (p *postgresStore) GetPgDB() (*gorm.DB, error) {
+	return gorm.Open(postgres.Open(
+		fmt.Sprintf("host=%v port=%v user=%v password=%v sslmode=disable",
+			config.Configuration.DBHost,
+			config.Configuration.DBPort,
+			config.Configuration.PGUserName,
+			config.Configuration.PGPassword),
+	))
```

Comment:
Any reason why the gorm.Config {SkipDefaultTransaction: true, and PrepareStmt: true} were not set, I saw they were used before.

Reply:
@peterlimg gorm's config is still used. 
This `db` client is only used to create `user` and `database`. After this, `db.DB().Close()` is called and then `db` is set to the one that you saw before.

```
func AutoMigrate(pgDB *gorm.DB) error {
	if err := createUser(pgDB); err != nil {
		return err
	}

	if err := createDB(pgDB); err != nil {
		return err
	}

	if err := grantPrivileges(pgDB); err != nil {
		return err
	}

	d, err := pgDB.DB()
	if err != nil {
		return err
	}

	if err := d.Close(); err != nil {
		return err
	}

	if err := datastore.GetStore().Open(); err != nil {
		return err
	}

	db := datastore.GetStore().GetDB()

	return migrateSchema(db)
}

```